README.md

```diff
@@ -28,6 +40,8 @@ $.ajax({
 ### Question 5
 In an AJAX POST request, what does the argument of the `.done()` callback - in the example below, that would be `data` - represent? (Hint: it's not the same as in Question 4.)
 
+In an ajax post request data would represent what you are sending to the API.
```

Comment:
Clarification here: in a POST request, `data` still represents what the server returns to you, but the convention is that the server will return whatever you sent.